internal/handlers/common/update.go

```diff
@@ -141,10 +141,7 @@ func UpdateDocument(doc, update *types.Document) (bool, error) {
 				}
 			}
 
-			setDocKeys := setDoc.Keys()
-			sort.Strings(setDocKeys)
```

Comment:
There are two other places where we call `sort.Strings`:

https://github.com/FerretDB/FerretDB/blob/09eb7ee5ca2f1313179a525610044e7d8ca6bb3d/internal/handlers/common/update.go#LL162C2-L162C13

https://github.com/FerretDB/FerretDB/blob/09eb7ee5ca2f1313179a525610044e7d8ca6bb3d/internal/handlers/common/update.go#L596

Should we update them too?